helpers/test-helper.js

```diff
@@ -114,6 +114,36 @@ export function web3GetRawCall(params) {
   });
 }
 
+export function stopMining() {
```

Comment:
Not used anywhere in the final version but indeed I tried to use them to eliminate the proximity calculations in finalPrice estimates. That didn't work because we have to forward time as well in between opening the auction, bidding and closing it :(